test/parallel_api/ranges/reduce_by_key.pass.cpp

```diff
@@ -0,0 +1,69 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include <oneapi/dpl/execution>
+#include <oneapi/dpl/algorithm>
+#include <oneapi/dpl/iterator>
+
+#include "support/test_config.h"
+#include "support/utils.h"
+
+#include <iostream>
+#include <vector>
+#include <cassert>
+
+#if TEST_DPCPP_BACKEND_PRESENT
+#include <CL/sycl.hpp>
+#endif // TEST_DPCPP_BACKEND_PRESENT
+
+int
+main()
+{
+#if TEST_DPCPP_BACKEND_PRESENT
```

Comment:
Please add this test case to https://github.com/oneapi-src/oneDPL/blob/c2b8326ff0dd1650820db8200ee7a074a3997b55/test/parallel_api/numeric/numeric.ops/reduce_by_segment.pass.cpp

Reply:
Done.